[CBRD-25386] Modified so that TBB is statically linked only to 'libcubrid.so' (#5581)
CUBRID/cubrid: 3rdparty/CMakeLists.txt

      INSTALL_COMMAND      make -j install
      "${LIBTBB_BYPRODUCTS}"
    )

> 
>     list(APPEND TBB_TARGETS ${LIBTBB_TARGET})

Reviewer: ```suggestion
    list(APPEND TBB_TARGETS ${LIBTBB_TARGET})
```

      INSTALL_COMMAND      make -j install
      "${LIBTBB_BYPRODUCTS}"
    )

>     # Append include directories and libraries for libcubrid.so only

Reviewer: 아래 주석과 중복된 설명으로 제거해도 될 것 같습니다.
```suggestion
```

Author: 제거하였습니다.

set (EP_LIBS ${EP_LIBS} PARENT_SCOPE)
set (EP_INCLUDES ${EP_INCLUDES} PARENT_SCOPE)

# TODO: EP_TARGETS, EP_LIBS, and EP_INCLUDES are currently linked across all targets.
# However, TBB is only required on the server side and should only be linked to libcubrid.so.
# To achieve this, TBB has been separated into a distinct list to ensure exclusive linking with libcubrid.so.
# Going forward, directories and libraries strictly used on the server side should be categorized

> # under the EP_SERVER_XXX section.

Reviewer: 주석을 남기기 보다는 차라리 이번에 EP_SERVER_* 형식으로 처리해 버리고, 주석을 지워버리죠.

set (EP_LIBS ${EP_LIBS} PARENT_SCOPE)
set (EP_INCLUDES ${EP_INCLUDES} PARENT_SCOPE)

> # [TODO]
> # EP_TARGETS, EP_LIBS, and EP_INCLUDES are linked everywhere.
> # Howerver, TBB is only used on the server side, so it should only be linked to libcubrid.so.
> # Therefore, we separated it into a TBB list to ensure it links exclusively to libcubrid.so.
> # In the future, please separate any directories and libraries that are exclusively used on the server side into EP_SERVER_XXX.

Reviewer 2: NIT:
```suggestion
# TODO: EP_TARGETS, EP_LIBS, and EP_INCLUDES are currently linked across all targets.
# However, TBB is only required on the server side and should only be linked to libcubrid.so.
# To achieve this, TBB has been separated into a distinct list to ensure exclusive linking with libcubrid.so.
# Going forward, directories and libraries strictly used on the server side should be categorized
# under the EP_SERVER_XXX section.
```
